src/nemo_evaluator/solvers/harbor.py

```
@@ -239,7 +239,9 @@ async def _download_agent_logs_inner(
             pass
 
 
-async def _patch_openhands_sdk(sandbox: "Sandbox", *, cmd_timeout: float | None = None) -> None:
+async def _patch_openhands_sdk(
+    sandbox: "Sandbox", *, cmd_timeout: float | None = None
```

Ruff (0.15.17)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)